webpack.dev.js

```diff
@@ -13,4 +13,9 @@ module.exports = merge(webpackCommon, {
       hmr: true,
     },
   },
+  plugins: [
+    new webpack.DefinePlugin({
+      API_URL: 'https://dev-kartat.hsldev.com',
```

Comment:
Hardcoded dev url in development. The Docker image will always be built with the production config.